kext/FileNVRAM/FileNVRAM.h

```diff
@@ -180,7 +182,7 @@ class FileNVRAM : public IODTNVRAM
     OSDictionary * mNvramMissDict;
     IOCommandGate* mCommandGate;
     OSString*      mFilePath;
-    IOTimerEventSource* mTimer;
+    IONotifier* disk_notifier_start_;
```

Comment:
Should I revert (I mean delete) all my commits to that one: [?](https://github.com/meklort/FileNVRAM/pull/3/commits/da513ff3ce7b4a5cca427b5e352131f077f61edf)